client/client.js

```diff
@@ -1,162 +1,153 @@
-         
-     });
-     client.editors[0].setShowPrintMargin(false);
-     client.editors[0].session.setMode("ace/mode/python");
-     client.editors[0].setValue(document.getElementById("pcapcode").innerHTML, -1);
- 
-     // streams editor
-     client.editors.push(ace.edit("STREAMS"));
- 
-     client.editors[1].setOptions({
-         enableBasicAutocompletion: true,
-         enableSnippets: true,
-         enableLiveAutocompletion: true
-         
-     });
-     client.editors[1].setShowPrintMargin(false);
-     client.editors[1].session.setMode("ace/mode/python");
-     client.editors[1].setValue(document.getElementById("streamscode").innerHTML, -1);
+    client.editors[0].setShowPrintMargin(false);
+    client.editors[0].session.setMode('ace/mode/python');
+    client.editors[0].setValue(document.getElementById('pcapcode').innerHTML, -1);
+
+    // streams editor
+    client.editors.push(ace.edit('STREAMS'));
+    client.editors[1].setOptions({
+        enableBasicAutocompletion: true,
+        enableSnippets: true,
+        enableLiveAutocompletion: true
+    });
+
+    client.editors[1].setShowPrintMargin(false);
+    client.editors[1].session.setMode('ace/mode/python');
+    client.editors[1].setValue(document.getElementById('streamscode').innerHTML, -1);
 }
 
 
 function create_terminals() {
-    
+
     Terminal.colors[256] = '#ffffff';
     Terminal.colors[257] = '#000000';
 
     client.terminals = {};
 
-    dim = get_console_size();
+    const { cols, rows } = get_console_size();
 
     client.terminals.console = new Terminal({
         screenKeys: true,
         useStyle: true,
-        cols: dim.cols,
-        rows: dim.rows,
+        cols,
+        rows,
     });
 
     client.terminals.code = new Terminal({
         screenKeys: true,
         useStyle: true,
-        cols: cols,
-        rows: rows,
+        cols,
+        rows,
     });
 
     client.terminals.tcpdump = new Terminal({
         screenKeys: true,
         useStyle: true,
-        cols: cols,
-        rows: rows,
+        cols,
+        rows,
     });
 
     Terminal.colors[256] = '#ffffff';
     Terminal.colors[257] = '#000000';
 
-    client.terminals.console.open(document.getElementById("console"));
-    client.terminals.code.open(document.getElementById("output"))
-    client.terminals.tcpdump.open(document.getElementById("tcpdump"))
+    client.terminals.console.open(document.getElementById('console'));
+    client.terminals.code.open(document.getElementById('output'));
+    client.terminals.tcpdump.open(document.getElementById('tcpdump'));
 
     // register some events
     client.terminals.console.on('data', function(data) {
-        client.socket.emit("console-input", data);
+        client.socket.emit('console-input', data);
 	});
 }
 
 
 function create_socket() {
     // Connect to the socket.io server
-    client.socket = io.connect('http://csi-trex-10:8080');
+    client.socket = io.connect('http://localhost:7171');

```

Comment:
124: I assume you are using `csi-trex-10` for localhost, this will trip some people up. Could alternatively put `127.0.0.1` here.